Services/TMS/TrainingSearch/classes/Helper.php

```diff
@@ -154,10 +152,24 @@ public function prepareModal(ilPropertyFormGUI $form)
 		$submit = $this->g_factory->button()->primary($this->g_lng->txt('search'), "#")->withOnLoadCode(function($id) use ($form_id) {
 			return "$('#{$id}').click(function() { $('#{$form_id}').submit(); return false; });";
 		});
- 
+
+		$reset = $this->g_factory->button()->standard($this->g_lng->txt('reset'), "#")->withOnLoadCode(function($id) use ($form_id) {
+			$dur1 = '$("input[name=\'f_duration[start]\']").val("'.date("01.01.Y").'");';
+			$dur2 = '$("input[name=\'f_duration[end]\']").val("'.date("31.01.Y").'");';
+			return "$('#{$id}').click(function() { 
+				$('#f_title').val('');
+				$('#f_type').val(-1);
+				$('#f_topic').val(-1);
+				$('#f_target').val(-1);
+				$('#f_not_min_member').prop('checked', false );
+				".$dur1."
+				".$dur2."
+				return false; 
+			});";
+		});
 
```

Comment:
$('#{$form_id}').reset();
??

Reply:
Ja habe ich auch vorher probiert. Aber wenn ich man sucht und dann wieder ins Form kommt, ist der Wert für den Reset was beim Rendern ausgewählt ist. Also die vorherigen Einträge.

Bsp. Man wählt unter Typ "Webinar" aus. Schickt die Suche ab. Ändert den Typ auf "Präsenz" und drückt dann Zurücksetzen, steht da wieder Webinar.

Reply:
ah, get it.